recipe/run_conda_forge_build_setup_osx

```diff
@@ -14,14 +14,16 @@ export MACOSX_DEPLOYMENT_TARGET="10.9"
 
 export CONDA_BUILD_SYSROOT="$(xcode-select -p)/Platforms/MacOSX.platform/Developer/SDKs/MacOSX${MACOSX_DEPLOYMENT_TARGET}.sdk"
 
-if [ -f "${CONDA_BUILD_SYSROOT}" ]
+if [ -d "${CONDA_BUILD_SYSROOT}" ]
 then
     echo "Found CONDA_BUILD_SYSROOT: ${CONDA_BUILD_SYSROOT}"
 else
     echo "Missing CONDA_BUILD_SYSROOT: ${CONDA_BUILD_SYSROOT}"
     exit 1
 fi
 
+ln -s ${CONDA_BUILD_SYSROOT} /opt/MacOSX10.9.sdk
```

Comment:
@jakirkham, what do you think about this?

Reply:
Probably works. Don't know if it would matter to use a symlink or a hardlink. Other thing to check would be how the rpaths or set (in case this might confuse matters there).

OTOH maybe it isn't worth the trouble and we should just hard code the whole value in `conda-forge-pinning`. FWIW this is what it would be ( https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/16 ).

Would also note that the new compilers pick up this setting by default anyways. Maybe nothing else needs them. Still it might make sense for `conda-build` to provide this default as well ( https://github.com/conda/conda-build/issues/2719 ).